protocol/communities/persistence.go

```diff
@@ -322,6 +322,59 @@ func (p *Persistence) SaveRequestToJoin(request *RequestToJoin) (err error) {
 	return err
 }
 
+func (p *Persistence) SaveRequestToJoinRevealedAddresses(request *RequestToJoin) (err error) {
+	tx, err := p.db.BeginTx(context.Background(), &sql.TxOptions{})
+	if err != nil {
+		return
+	}
+	defer func() {
+		if err == nil {
+			err = tx.Commit()
+			return
+		}
+		// don't shadow original error
+		_ = tx.Rollback()
+	}()
+
+	query := `INSERT OR REPLACE INTO communities_requests_to_join_revealed_addresses (request_id, address, signature) VALUES (?, ?, ?)`
```

Comment:
I don't think saving the signature is useful. Once we have confirmed that the addresses belong to the account, I think we can just save the addresses? Maybe I forgot a use case though.